test/benchmarks/Performance.StackExchange.Redis/Performance.StackExchange.Redis.csproj

```diff
@@ -12,7 +12,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\tools\Datadog.Core.Tools\Datadog.Core.Tools.csproj" />
+    <ProjectReference Include="..\..\..\tools\Datadog.Core.Tools\Datadog.Core.Tools.csproj" />
```

Comment:
I think we can delete `Performance.StackExchange.Redis`. It's not used anywhere and is superseded by `/test/benchmarks/Benchmarks.Trace/RedisBenchmark.cs`.